.gitlab-ci.yml

```diff
@@ -100,6 +79,34 @@ SonarQube:
     INTERNAL_NETWORK: "true"
     GIT_DEPTH: 0
 
+Manifests:
+  stage: test
+  extends: .terraform
+  script:
+    - schutzbot/deploy.sh
+    - sudo schutzbot/manifest_tests.sh
+  artifacts:
+    when: always
+    paths:
+      - manifest-db/generated-image-infos/
+  parallel:
+    matrix:
+      - RUNNER:
+          - aws/fedora-35-x86_64
+          - aws/fedora-35-aarch64
```

Comment:
We are not testing on F36. I know there eis a problem with the OSTree commit but I don't think we should not test on F36 at all.

Reply:
F36 is not yet ready. We need a DB update for that. We need to merge the PR on manifest-db and run a update. Then I can activate the runners for F36.

Reply:
I"ve added them.